code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -32,6 +33,36 @@
 		GLOB.pre_setup_antags += M.mind
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//               SCAMPS                     //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/scamp
```

Comment:
this ruleset needs to stop being picked once pop reaches a certain point because it has way too much weight